EIPS/eip-1109.md

```diff
@@ -0,0 +1,54 @@
+---
+eip: 1109
+title: Remove CALL costs for precompiled contracts
+author: Jordi Baylina (@jbaylina)
+discussions-to: jordi@baylina.cat
+status: Draft
+type: Core
```

Comment:
You need to supply a category as well as a type; see EIP 0 or EIP-X for guidance.